### PR TITLE
Testing/deployui

### PR DESCRIPTION
**Pull request by gboie**
Updated requirements.txt with the latest. This is also design to get a Vercel URL.

### CODE REVIEW

**Comment by khdegraaf** [comment]
But you are getting a Vercel UI link like I said.

File: cmd/ebrelayer/relayer/ethereum.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ import (
 
 const (
 	transactionInterval = 10 * time.Second
-	trailingBlocks      = 50
+	trailingBlocks      = 4
```
Comment (intl-man) [inline cmd/ebrelayer/relayer/ethereum.go:43]: Why are you changing the relayer?

File: cmd/ebrelayer/txs/relayToEthereum.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ import (
 const (
 	// GasLimit the gas limit in Gwei used for transactions sent with TransactOpts
 	GasLimit            = uint64(500000)
-	transactionInterval = 60 * time.Second
+	transactionInterval = 3 * time.Second
```
Comment (intl-man) [inline cmd/ebrelayer/txs/relayToEthereum.go:23]: I don't believe we should be changing this.

File: test/integration/FUTURES.txt [hunk 3]
```diff
@@ -0,0 +1,83 @@
+Ethereum
```
Comment (intl-man) [inline test/integration/FUTURES.txt:1]: Can you please re-write it in markdown?

File: test/integration/ethereum_keys.json [hunk 4]
```diff
@@ -0,0 +1 @@
...0186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"}}},"private_keys":{"0x627306090abab3a6e1400e9345bc60c78a8bef57":"c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3","0xf17f52151ebef6c7334fad080c5704d77216b732":"ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f","0xc5fdf4076b8f3a5357c5e395ab970b5b54098fef":"0dbbe8e4ae425a6d2687f1a7e3ba17bc98c673636790f1b8ad91193c05875ef1","0x821aea9a577a9b44299b9c15c88cf3087f3b5544":"c88b703fb08cbea894b6aeff5a544fb92e78a18e19814cd85da83b71f772aa6c","0x0d1d4e623d10f9fba5db95830f7d3839406c6af2":"388c684f0ba1ef5017716adb5d21a053ea8e90277d0868337519f97bede61418","0x2932b7a2355d6fecc4b5c0b6bd44cc31df247a2e":"659cbb0e2411a44db63778987b1e22153c086a95eb6b18bdf89de078917abc63","0x2191ef87e392377ec08e7c08eb105ef5448eced5":"82d052c865f5763aad42add438569276c00d3d88a2d062d36b2bae914d58b8c8","0x0f4f2ac550a1b4e2280d04c21cea7ebd822934b5":"aa3680d5d48a8283413f7a108367c7299ca73f553735860a87b08f39395618b7","0x6330a553fc93768f612722bb8c2ec78ac90b3bbc":"0f62d96d6675f32685bbdb8ac13cda7c23436f63efbb9d07700d8669ff12b7c4","0x5aeda56215b167893e80b4fe645ba6d5bab767de":"8d5366123cb560bb606379f90a0bfd4769eecc0557f1b362dcae9012b548b1e5"}}
```
Comment (intl-man) [inline test/integration/ethereum_keys.json:1]: New line.

**Review by intl-man — CHANGES_REQUESTED** [review]

**Pull request closed without merging**